src/components/PeriodicReports/ReportLabel.tsx

```diff
@@ -25,7 +37,16 @@ export const ReportLabel = ({
   if (!rep) {
     return null;
   }
-  return <>{getLabel(rep.start, rep.end)}</>;
+
+  const isSkippedProgressReport =
+    rep.__typename === 'ProgressReport' && rep.skippedReason.value;
+
+  return (
+    <div className={classes.label}>
```

Comment:
I think span might be more appropriate.
```suggestion
    <span className={classes.label}>
```